gamestonk_terminal/cryptocurrency/__init__.py

```diff
@@ -0,0 +1,208 @@
+import argparse
```

Comment:
I didn't know where to put it as it's common for all sources. I think first ideas was to put it in helpers, but I had some kind of conflict with imports. But will take a look at the evening :D

Reply:
Yea so there is import config if I put it in `cryptocurrency_helper`, because my views `load` and `find` are using `pycoingecko_view`, `binance_view`, `coinpaprika_view` and in the same time those `pycoingecko_view`, `binance_view`, `coinpaprika_view`  have imports from `cryptocurrency_helper`. I ve never had this kind of clutch with imports, and I am thinking how to deal with that in a proper way.

Reply:
This definitely should not be in ```__init__```

Reply:
That solution sounds perfect to me! 

crypto_helpers should be used to hold the main commands associated with crypto_controller. Then having an addition dataframe_helpers seems the right approach to me :)